src/cli-util.js

```diff
@@ -33,6 +34,21 @@ function getOptions(argv) {
   };
 }
 
+function getRequirePragmaOption(argv) {
+  const value = argv["require-pragma"];
+
+  if (value === undefined) {
+    return value;
+  }
+
+  // if the user provided a value with `@` in the beginning anyway, strip it off.
+  if (value.startsWith("@")) {
+    return value.substr(1);
+  }
```

Comment:
Please don't silently drop things. Instead you want to throw an exception with a clear error message.

Reply:
Actually, it seems to be confusing so why not require people to add @ in the command line.

```
--pragma @format
```